src/Workspaces/Core/Portable/CodeFixes/CodeFixContext.cs

```diff
@@ -48,7 +48,13 @@ public struct CodeFixContext
         /// </summary>
         /// <param name="document">Document to fix.</param>
         /// <param name="span">Text span within the <paramref name="document"/> to fix.</param>
-        /// <param name="diagnostics">Diagnostics to fix. All the diagnostics should have the same span <paramref name="span"/>.</param>
+        /// <param name="diagnostics">
+        /// Diagnostics to fix. All the diagnostics should have the same span <paramref name="span"/>.
```

Comment:
Remove the first "span" so that the "span" in the paramref finishes the sentence. Otherwise it'll look like a repeated word when displayed in Intellisense.